extensions/data-transfer/portability-data-transfer-google/src/test/java/org/datatransferproject/datatransfer/google/mediaModels/GoogleMediaItemTest.java

```diff
@@ -0,0 +1,132 @@
+package org.datatransferproject.datatransfer.google.mediaModels;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Date;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Optional;
+import org.datatransferproject.types.common.models.photos.PhotoModel;
+import org.datatransferproject.types.common.models.videos.VideoModel;
+import org.junit.Test;
+public class GoogleMediaItemTest {
+
+  @Test
+  public void getMimeType_photoModel_mimeTypeFromFilename() throws Exception {
+    GoogleMediaItem photoMediaItem = getPhotoMediaItem();
+    Map<String, String> filenameToMimeTypeMap = Map.of(
+        "file.jpg", "image/jpeg",
+        "file.png", "image/png",
+        "file.apng", "image/apng",
```

Comment:
FYI it looks like the gradle test is failing on this file type: 

```
GoogleMediaItemTest > getMimeType_photoModel_mimeTypeFromFilename FAILED
    org.opentest4j.AssertionFailedError: expected: <image/apng> but was: <image/vnd.mozilla.apng>
        at org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)
        at org.junit.jupiter.api.AssertionUtils.failNotEqual(AssertionUtils.java:62)
        at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)
        at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)
        at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:1141)
        at org
```

Reply:
Okay, I believe I found the issue here. According to the docs, probeContentType ["uses the installed FileTypeDetector implementation to probe the given file to determine its content type."](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#probeContentType-java.nio.file.Path-)

Locally, my installed FileTypeDetector returns "image/apng" for 'file-name.apng', but it seems like in our pipeline we might have a different FileTypeDetector? I am not sure where this FileTypeDetector is coming from, maybe it depends on the OS running the pipeline? Or perhaps it is a Gradle specified detail? I can't seem to find more info on this, but I think the most important part of this is that it indicates that we must decide on a mimeType conversion convention to prescribe to.

Reply:
Instead of focusing on testing the probeContentType implementation, i think its fine to focus on testing the fallback logic. so i think its fine to remove this file type from the test. 

WDYT?

Reply:
Makes sense to me! I do think we should make note of this though, since it feels like indeterministic code. On one machine the mimetype may be x, and another it may be y.